validation/subscription_operations_test.go

```
@@ -0,0 +1,399 @@
+package validation
```

golangci-lint (2.12.2)

[error] ST1000: package comment should be of the form "Package core..."

(staticcheck)